lib/registration.pm

```diff
@@ -85,7 +85,8 @@ sub accept_addons_license {
     #   isc co SUSE:SLE-15:GA 000product
     #   grep -l EULA SUSE:SLE-15:GA/000product/*.product | sed 's/.product//'
     # All shown products have a license that should be checked.
-    my @addons_with_license = qw(geo live rt idu ids lgm ses);
+    # On SLE 15, ha and we have license but live patching doesn't have.
```

Comment:
This is probably product bug ...

Reply:
Thanks, so live patching should have license on SLE15+ also, I will updated code.

Reply:
File a new bug for ha,we don't have license on proxySCC, bsc#1116657. This PR will fix after the bug fixed, or will affect testcases run on proxySCC.